src/components/MapView/Analyser/index.tsx

```diff
@@ -350,10 +443,10 @@ function Analyser({ classes }: AnalyserProps) {
                 className={classes.innerAnalysisButton}
                 onClick={runAnalyser}
                 disabled={
-                  !!thresholdError || // if there is a threshold error
-                  !selectedDate || // or date hasn't been selected
-                  !hazardLayerId || // or hazard layer hasn't been selected
-                  !baselineLayerId || // or baseline layer hasn't been selected
+                  !!form.thresholdError || // if there is a threshold error
```

Comment:
It would be nice to have a isValid function to decide if the form is valid to submit, then this could be used both here and when the analysis is dispatched.

Reply:
Done